src/webui/components/Footer/index.js

```diff
@@ -1,63 +1,41 @@
-import React, {Component} from 'react';
+/**
+ * @prettier
+ * @flow
+ */
 
-import classes from './footer.scss';
-import logo from './logo.svg';
-import earth from './earth.svg';
+import React from 'react';
+import type { Element } from 'react';
 
-// Vectors from Twitter Emoji (Open Source)
-import brazilFlag from './flags/brazil-1f1e7-1f1f7.svg';
-import chinaFlag from './flags/china-1f1e8-1f1f3.svg';
-import indiaFlag from './flags/india-1f1ee-1f1f3.svg';
-import nicaraguaFlag from './flags/nicaragua-1f1f3-1f1ee.svg';
-import pakistanFlag from './flags/pakistan-1f1f5-1f1f0.svg';
-import spainFlag from './flags/spain-1f1ea-1f1f8.svg';
+import { version } from '../../../../package.json';
+import { Wrapper, Left, Right, Earth, Flags, Love, Flag, Logo, Inner, ToolTip } from './styles';
 
-export default class Footer extends Component {
-  constructor(props) {
-    super(props);
-    this.handleEarthIconClick = this.handleEarthIconClick.bind(this);
-    this.state = {
-      showAuthorsGeographic: false
-    };
-  }
+const Footer = (): Element<Wrapper> => (
+  <Wrapper>
+    <Inner>
+      <Left>
+        Made with
+        <Love>♥</Love>
+        on
+        <ToolTip>
+          <Earth name="earth" />
+          <Flags>
+            <Flag name="spain" />
+            <Flag name="nicaragua" />
+            <Flag name="india" />
+            <Flag name="brazil" />
+            <Flag name="pakistan" />
+            <Flag name="china" />
+            <Flag name="austria" />
+          </Flags>
+        </ToolTip>
+      </Left>
+      <Right>
+        Powered by
+        <Logo name="verdaccio" size="md" pointer img onClick={() => window.open('http://www.verdaccio.org/', '_blank')} />
```

Comment:
Better to move Verdaccio URL to https://github.com/verdaccio/verdaccio/blob/4.x/src/lib/constants.js

Reply:
yes..I will 👍

Reply:
I was thinking..I don't want to mix webui with lib...the webui has an utils folder and lib has another one....we should create some constant file in webui as well...I will think about something.